src/AppBundle/Entity/ProcurationRequest.php

```diff
@@ -341,7 +356,7 @@ public function validateElectionsChosen(ExecutionContextInterface $context)
         $context->addViolation('Vous devez choisir au moins une élection');
     }
 
-    public function importAdherentData(Adherent $adherent)
+    public function importAdherentData(Adherent $adherent): void
```

Comment:
Please try to make other commits for cs fixes to ease the review with a thinner diff.